[2025-02-09T18:52:32.434+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T18:52:32.455+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_postgres_and_csv_data.run_load_data_script manual__2025-02-09T18:46:22.502200+00:00 [queued]>
[2025-02-09T18:52:32.465+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_postgres_and_csv_data.run_load_data_script manual__2025-02-09T18:46:22.502200+00:00 [queued]>
[2025-02-09T18:52:32.467+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-02-09T18:52:32.618+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): run_load_data_script> on 2025-02-09 18:46:22.502200+00:00
[2025-02-09T18:52:32.627+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=208) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T18:52:32.631+0000] {standard_task_runner.py:72} INFO - Started process 210 to run task
[2025-02-09T18:52:32.630+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_postgres_and_csv_data', 'run_load_data_script', 'manual__2025-02-09T18:46:22.502200+00:00', '--job-id', '286', '--raw', '--subdir', 'DAGS_FOLDER/extract_postgres_and_csv_data.py', '--cfg-path', '/tmp/tmpy4f1htqt']
[2025-02-09T18:52:32.632+0000] {standard_task_runner.py:105} INFO - Job 286: Subtask run_load_data_script
[2025-02-09T18:52:32.727+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_postgres_and_csv_data.run_load_data_script manual__2025-02-09T18:46:22.502200+00:00 [running]> on host 58bc83bc12fd
[2025-02-09T18:52:32.917+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_postgres_and_csv_data' AIRFLOW_CTX_TASK_ID='run_load_data_script' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T18:46:22.502200+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T18:46:22.502200+00:00'
[2025-02-09T18:52:32.919+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-09T18:52:32.919+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-09T18:52:32.920+0000] {logging_mixin.py:190} INFO - Current task name:run_load_data_script state:running start_date:2025-02-09 18:52:32.456609+00:00
[2025-02-09T18:52:32.921+0000] {logging_mixin.py:190} INFO - Dag name:extract_postgres_and_csv_data and current dag run status:running
[2025-02-09T18:52:32.921+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T18:52:32.923+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-09T18:52:32.925+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/meltano/meltano_project/load/load_data_csv_for_db.py']
[2025-02-09T18:52:32.940+0000] {subprocess.py:99} INFO - Output:
[2025-02-09T18:52:36.515+0000] {subprocess.py:106} INFO - Conexão bem-sucedida ao banco de dados!
[2025-02-09T18:52:36.516+0000] {subprocess.py:106} INFO - Tabela 'order_details' criada com sucesso!
[2025-02-09T18:52:36.517+0000] {subprocess.py:106} INFO - Carregando dados do arquivo: /opt/***/meltano/data/csv/2025-02-09/order_details.csv
[2025-02-09T18:52:36.518+0000] {subprocess.py:106} INFO - Conexão bem-sucedida ao banco de dados!
[2025-02-09T18:52:36.519+0000] {subprocess.py:106} INFO - Dados inseridos com sucesso!
[2025-02-09T18:52:36.520+0000] {subprocess.py:106} INFO - Conexão bem-sucedida ao banco de dados!
[2025-02-09T18:52:36.521+0000] {subprocess.py:106} INFO - Erro ao buscar dados: No module named 'openpyxl'
[2025-02-09T18:52:36.699+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-09T18:52:36.763+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T18:52:36.766+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_postgres_and_csv_data, task_id=run_load_data_script, run_id=manual__2025-02-09T18:46:22.502200+00:00, execution_date=20250209T184622, start_date=20250209T185232, end_date=20250209T185236
[2025-02-09T18:52:36.799+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-09T18:52:36.800+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-09T18:52:36.801+0000] {logging_mixin.py:190} INFO - Dag name:extract_postgres_and_csv_data queued_at:2025-02-09 18:46:22.525077+00:00
[2025-02-09T18:52:36.802+0000] {logging_mixin.py:190} INFO - Task hostname:58bc83bc12fd operator:BashOperator
[2025-02-09T18:52:36.860+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T18:52:36.901+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-09T18:52:36.904+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
